boards/arm/serpente/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+# SPDX-License-Identifier: Apache-2.0
+
+#zephyr_library_sources(board_init.c)
```

Comment:
Drop the commented-out line

Reply:
Good catch. Not sure how I missed that when committing. The line is actually needed to properly setup the onboard led. Otherwise it would be glowing blue as a leftover from SAM-BA bootloader.